great_expectations/core/usage_statistics/anonymizers/anonymizer.py

```diff
@@ -118,17 +133,29 @@ def _is_core_great_expectations_class(class_name: str) -> bool:
         return class_name.startswith(Anonymizer.CORE_GE_OBJECT_MODULE_PREFIX)
 
     @staticmethod
-    def _is_parent_class_recognized(
-        classes_to_check,
-        object_=None,
-        object_class=None,
-        object_config=None,
+    def _get_parent_class(
```

Comment:
I wanted to change this name because `_is` denotes a boolean return type. Since we have an `Optional[str]`, I think `_get` makes more sense?